.vscode/settings.json

```diff
@@ -77,7 +77,12 @@
 		"backend/payload/*": true,
 		"**/yarn-error.log": true
 	},
-	"cSpell.words": ["Blocklist", "interactable"],
+	"cSpell.words": [
+		"autoinject",
```

Comment:
I love how easy the plugin makes it to ignore certain words:

![Screenshot 2023-04-21 at 1 11 58 PM](https://user-images.githubusercontent.com/58678/233716330-bb4eccd5-4c78-491b-a5e6-193fd7270d7e.png)